### PR TITLE
Write bylaws

### PR DESCRIPTION
**Pull request by rrlapointe**
Continue writing bylaws

### CODE REVIEW

**Comment by rrlapointe** [comment]
I found another problem.

File: board-of-directors.tex [hunk 1]
```diff
@@ -28,21 +28,21 @@ \section{Number, Election, and Tenure}
...
+Theta Pi shall be ex officio Directors of the Corporation during the times such
+persons occupy their offices as President, Treasurer, and House Manager,
+respectively, of the Eta Tau Chapter of Beta Theta Pi. These three Directors are
+referred to as `ex officio Directors' in these Bylaws.
 
 In order to be elected and in order to hold office, each elected Director must
 be a person who was conferred a Bachelor's Degree or Master's Degree by
-Worcester Polytechnic Institute, and at the time so conferred was an initiated
-Brother in good standing of the Eta Tau Chapter of Beta Theta Pi Fraternity.
+Worcester Polytechnic Institute, and at the time so conferred was either an
```
Comment (ArchdukeTim) [inline board-of-directors.tex:43]: I wonder if we shouldn't simply limit it to those in good standing with the chapter. I assume the default standing of anyone is "good", and through Kai, or just a censure for those not in the chapter, one would go to not in good standing

**Review by ArchdukeTim — COMMENTED** [review]
When we adopt these will we discuss them? Or will it be an all or nothing kinda deal?

File: board-of-directors.tex (hunk 1 above)
Comment (rrlapointe) [reply board-of-directors.tex:43]: Your assumption sounds correct to me. I deliberately worded this so it depends on you having been in good standing at the time that you graduated, not currently. That prevents the Chapter from being able to manipulate who is eligible by declaring alumni to be not in good standing. They would have to produce evidence that you weren't in good standing when you graduated. A good example of _not_ in good standing would be if someone was on suspension at the time that he graduated. That would permanently bar him from being a Director.

**Comment by rrlapointe** [comment]
The bylaws can be amended by a majority of the Directors. An amendment can be passed at any Board of Directors meeting. So after we adopt them, we could amend them.

**Review by ArchdukeTim — APPROVED** [review]

**Pull request merged**